tests/controllers/user.test.js

```diff
@@ -118,11 +119,13 @@ test("user verify request with invalid token returns error", async () => {
 });
 
 
-test("user unsubscribe GET request with valid token returns error", async () => {
-  const validToken = "0e2cb147-2041-4e5b-8ca9-494e773b2cf0";
+test("user unsubscribe GET request with valid token and hash returns 200 without error", async () => {
+  // from db/seeds/test_subscribers.js
+  const subscriberToken = "0e2cb147-2041-4e5b-8ca9-494e773b2cf1";
+  const subscriberHash = getSha1("firefoxaccount@test.com");
```

Comment:
I had to do a bit of scrolling to verify these looked the same.
Not sure if it's worth the hassle to possibly export the dummy user from db/seeds/test_subscribers.js to avoid having to duplicate the UUIDs in 2+ places.

Reply:
Good idea. I made `test_subscribers.js` export `TEST_DATA` for use in tests.